zetaclient/chains/bitcoin/observer/inbound.go

```diff
@@ -421,6 +421,18 @@ func GetBtcEvent(
 				return nil, nil
 			}
 
+			// switch to depositor fee V2 if
+			// 1. it is bitcoin testnet, or
+			// 2. it is bitcoin mainnet and upgrade height is reached
+			// TODO: remove CalcDepositorFeeV1 and below conditions after the upgrade height
```

Comment:
Seems we can create an issue for the TODO?

Reply:
https://github.com/zeta-chain/node/pull/2765/commits/b0447c29c974a47e424fb1a60283cb6557c3101d